backend/app/services/adapters/team_kinds.py

```
@@ -2309,5 +2364,176 @@ def copy_team(
+        Skills are stored in the ghost's spec, not the bot's spec.
+        """
+        # Look up the ghost that holds skill_refs
+        bot_spec = bot.json.get("spec", {})
+        ghost_ref = bot_spec.get("ghostRef", {})
+        ghost = (
+            db.query(Kind)
+            .filter(
+                Kind.kind == "Ghost",
+                Kind.name == ghost_ref.get("name"),
+                Kind.namespace == ghost_ref.get("namespace", bot.namespace),
+                Kind.is_active == True,
```

Ruff (0.15.12)

[error] Avoid equality comparisons to `True`; use `Kind.is_active:` for truth checks

Replace with `Kind.is_active`

(E712)